prow/plugins/approve/approvers/owners.go

```diff
@@ -670,6 +672,12 @@ Associated issue requirement bypassed by:{{range $index, $approval := .ap.ListNo
 
 The full list of commands accepted by this bot can be found [here]({{ .commandHelpLink }}?repo={{ .org }}%2F{{ .repo }}).
 
+{{ if len .staleRepositoryState -}}
```

Comment:
I don't think this should be part of the approval message. This is shown to everyone who creates a PR but most of them don't have anything to do with managing OWNERS files.

I would suggest to instead build a CLI that allows to find stale entries.

Reply:
there is an outside chance that this will help a random PR-author by warning them that the nominal OWNER they had pinged for review was actually no longer an active collaborator.  And really, I would hope that shortly after this list got any entries, the active maintainers would prune `OWNERS` and future approval messages would be clean.  So I still like this. 
 But since I expect neither of those arguments to convince anyone:
 
 /close
 
 I'll happily re-open if I'm wrong and there is renewed interest.